libs/features/search/src/ui/views/search-modal/search-modal.component.ts

```diff
@@ -62,11 +70,37 @@ export class SearchModalComponent {
       return filters;
    }
 
+   processQuery(event: string)
```

Comment:
TODO: in the future, we will create solution the would not involve the use if the subscribes.   
** TECH DEBT